identity.go

```diff
@@ -31,3 +31,15 @@ func (c *IdentityController) List(ctx *app.ListIdentityContext) error {
 		return ctx.OK(result)
 	})
 }
+
+// Show runs the show action.
+func (c *IdentityController) Show(ctx *app.ShowIdentityContext) error {
```

Comment:
Sure. I assume you mean can you rename the Go Type object. It's not yet converted to jsonapi so it doesn't have a api type yet.